test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt

```diff
@@ -4,6 +4,7 @@
 
 
 
+
```

Comment:
It would be great if we could clean up the kickstart to avoid this whitespace, but won't block on it.